src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1668,26 +1668,23 @@ private DiagnosticInfo GetBadEventUsageDiagnosticInfo(EventSymbol eventSymbol)
                 new CSDiagnosticInfo(ErrorCode.ERR_BadEventUsageNoField, leastOverridden);
         }
 
-        internal static bool AccessingAutoPropertyFromConstructor(BoundPropertyAccess propertyAccess, Symbol fromMember)
+        internal static bool IsPropertyAssignableFromConstructor(BoundPropertyAccess propertyAccess, Symbol fromMember)
         {
-            return AccessingAutoPropertyFromConstructor(propertyAccess.ReceiverOpt, propertyAccess.PropertySymbol, fromMember);
+            return IsPropertyAssignableFromConstructor(propertyAccess.ReceiverOpt, propertyAccess.PropertySymbol, fromMember);
         }
 
-        private static bool AccessingAutoPropertyFromConstructor(BoundExpression receiver, PropertySymbol propertySymbol, Symbol fromMember)
+        private static bool IsPropertyAssignableFromConstructor(BoundExpression receiver, PropertySymbol propertySymbol, Symbol fromMember)
```

Comment:
>IsPropertyAssignableFromConstructor [](http://example.com/codeflow?start=28&length=35)

The name sounds too general. In particular, it sounds as if it gives an answer for any constructor. In fact, I think, the intent is to determine that, in the given context (not in some constructor), an access to the property is equivalent to an access to the backing field. It also is not about assignability in a general sense, a regular property with a setter is assignable pretty much anywhere, but this helper is going to return false for it. So, I would drop "Assignable" and "FromConstructor" from the name. The context is provided either by the Binder, or by the ```fromMember``` parameter, there is nothing we need to say in the name about that, I think.